sky/clouds/service_catalog/os_image/README.md

```diff
@@ -0,0 +1,17 @@
+# Custom OS Image Generation for SkyPilot
+
+## GCP
+The following is an example of generating a custom skypilot image for GPU instance running Debian 11 OS.
+1. Use SkyPilot to launch a VM \
+`sky launch sky/clouds/os_image/skypilot-gcp-gpu-debian-11.yaml`
+
+2. Stop the VM and create a custom image with name *skypilot-gcp-gpu-debian-11-v{YYMMDD}* \
+`gcloud compute images create ${IMAGE_NAME} --source-disk=${VM_BOOT_DISK_NAME} --source-disk-zone=${VM_ZONE} --family=skypilot-gpu --storage-location=us`
+
+3. Make image public so that any authenticated GCP user can use it \
+`gcloud compute images add-iam-policy-binding ${IMAGE_NAME} --member='allAuthenticatedUsers' --role='roles/compute.imageUser'`
```

Comment:
````suggestion
3. Make image public so that any authenticated GCP user can use it
```bash
gcloud compute images add-iam-policy-binding ${IMAGE_NAME} --member='allAuthenticatedUsers' --role='roles/compute.imageUser'
```
````